src/DssSpell.sol

```diff
@@ -58,35 +58,50 @@ contract DssSpellAction is DssAction {
 
```

Comment:
I think `OsmAbstract` (L24) can be removed as not used in the spell

Reply:
Removed!